### PR TITLE
WAVE2-836 - Update Drupal to version 7.53

### PR DESCRIPTION
**Pull request by SandyVallee**
Drupal core switched from 7.52 to 7.53.

### CODE REVIEW

File: robots.txt [hunk 1]
```diff
@@ -1,10 +1,20 @@
-User-Agent: *
```
Comment (dpfitzsi) [inline robots.txt]: Replacing robots.txt file with the default could be an issue. Will verify this with Geoff.

File: modules/update/update.module [hunk 2]
```diff
@@ -520,37 +520,25 @@ function _update_get_cached_available_releases() {
...
+    $message['body'][] = t('You can automatically install your missing updates using the Update manager:', array(), array('langcode' => $langcode)) . "\n" . url('admin/reports/updates/update', array('absolute' => TRUE, 'language' => $language));
+  }
   $settings_url = url('admin/reports/updates/settings', array('absolute' => TRUE));
   if (variable_get('update_notification_threshold', 'all') == 'all') {
     $message['body'][] = t('Your site is currently configured to send these emails when any updates are available. To get notified only for security updates, !url.', array('!url' => $settings_url));
   }
   else {
     $message['body'][] = t('Your site is currently configured to send these emails only when security updates are available. To get notified for any available updates, !url.', array('!url' => $settings_url));
   }
-  if ($core) {
```
Comment (dpfitzsi) [inline modules/update/update.module]: Looks like our previous build of 7.52 may have been done via the Pantheon dashboard. There are a few Pantheon specific functions that are going to be deleted in this update.

File: ucalgary.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Warning: Permanently added '[appserver.live.c06645d8-1b68-4ea5-a868-bf43766ae963.drush.in]:2222,[162.242.168.114]:2222' (RSA) to the list of known hosts.
```
Comment (dpfitzsi) [inline ucalgary.sql]: Should be removed.

File: .gitignore [hunk 4]
```diff
@@ -1,51 +1,6 @@
-# Ignore paths that contain user-generated content.
```
Comment (dpfitzsi) [inline .gitignore]: I will verify whether we should change this file or not. Likely we want to keep the version that is currently in GitHub and remove whatever Drupal is loading here.

File: .idea/modules.xml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dpfitzsi) [inline .idea/modules.xml]: File should be deleted.

File: .idea/vcs.xml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dpfitzsi) [inline .idea/vcs.xml]: File should be deleted.

File: .idea/workspace.xml [hunk 7]
```diff
@@ -0,0 +1,441 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dpfitzsi) [inline .idea/workspace.xml]: Should be deleted.

File: .idea/www.ucalgary.iml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dpfitzsi) [inline .idea/www.ucalgary.iml]: Should be deleted.

**Comment by dpfitzsi** [comment]
Here is the link to the Pantheon version of 7.53 which should mean fewer file changes.

https://github.com/pantheon-systems/drops-7/releases/tag/7.53

**Comment by dpfitzsi** [comment]
Looking much better.  A couple more changes to make this clean:

Replace:
robots.txt
.gitignore

Remove:
sites/default/local_database.inc
.idea folder
ucalgary.sql

File: robots.txt (hunk 1 above)
Comment (SandyVallee) [inline robots.txt]: done

File: .idea/modules.xml (hunk 5 above)
Comment (SandyVallee) [inline .idea/modules.xml]: done

File: .idea/www.ucalgary.iml (hunk 8 above)
Comment (SandyVallee) [inline .idea/www.ucalgary.iml]: done

**Pull request merged**